FIPS/src/test/java/net/snowflake/client/jdbc/ConnectionFipsIT.java

```diff
@@ -33,6 +33,7 @@
 import org.junit.experimental.categories.Category;
 
 @Category(TestCategoryFips.class)
+@ConditionalIgnoreRule.ConditionalIgnore(condition = RunningOnGithubAction.class)
```

Comment:
This is relating to [SNOW-571680](https://snowflakecomputing.atlassian.net/browse/SNOW-571680) where FIPS tests are failing. Seems like jenkins configuration issue since nothing changed on JDBC side.